rasa/nlu/test.py

```diff
@@ -1820,7 +1824,7 @@ def compute_metrics(
     )
 
 
-def compare_nlu(
+async def compare_nlu(
```

Comment:
This is also called from [here](https://github.com/RasaHQ/rasa/blob/bc76915a8fbd40f505f75c8246f03f26caaa0323/rasa/test.py#L220). I suggest to also add a `rasa.utils.common.run_in_loop` there